Engineering/23-fosscopyright.md

```diff
@@ -16,8 +16,16 @@ Licenses which only permit non-commercial redistribution or modification of the
 - [Eclipse Public License (EPL)](https://en.wikipedia.org/wiki/Eclipse_Public_License)
 - [MIT License](https://en.wikipedia.org/wiki/MIT_License)
 
-## Copyright
+## What is Copyright?
 
 Copyright is a type of intellectual property that gives its owner the exclusive right to make copies of a creative work, usually for a limited time. The creative work may be in a literary, artistic, educational, or musical form. Copyright is intended to protect the original expression of an idea in the form of a creative work, but not the idea itself. A copyright is subject to limitations based on public interest considerations. It is widely believed that copyrights are a must to foster cultural diversity and creativity.
 
-Copyrights can be granted by public law and are in that case considered "territorial rights". This means that copyrights granted by the law of a certain state, do not extend beyond the territory of that specific jurisdiction. Copyrights of this type vary by country; many countries, and sometimes a large group of countries, have made agreements with other countries on procedures applicable when works "cross" national borders or national rights are inconsistent.
+## Why Copyright is bad?
+
+Copyright is acceptable when it is quantifiable but many products are abstract like music, digital media. So keeping the contract fair and simple is pretty hard. Copyright is bad in terms of creativity and freedom. Because of copyright, people are restricted and their hands are tied to avoid any overlap with others. The problem turns out to be the fact that the central features of copyright are polar opposite to the things that support creativity.
+
+Creativity is a tricky thing to understand, and more trickier to understand what animates the creative spark and why some people are more creative than others. But when creativity is quantified in money because of the ownership of that creative idea, creativity loses traction.
```

Comment:
This needs changed for the same reason as above, claims that copyright is only bad is a problem. There are also multiple grammar issues, I'm fixing those in this suggestion for your reference, but I suspect these two lines will need re-written outright.
```suggestion
Creativity is complicated, more so when you try to understand what motivates the creative spark or why some people are more creative than others. When creativity is rewarded monetarily because of the ownership of that creative idea, creativity loses traction.
```